crates/sdk/src/keygen/mod.rs

```diff
@@ -438,6 +440,14 @@ impl AggProvingKey {
             wrapper,
             profiling: halo2_config.profiling,
         };
+
+        tracing::info!(
+            "agg_keygen wall time: {:.3}s",
+            wall_timer.elapsed().as_secs_f64()
+        );
+        #[cfg(feature = "bench-metrics")]
+        metrics::gauge!("total_proof_time_ms").set(wall_timer.elapsed().as_millis() as f64);
```

Comment:
this is weird that total_proof_time_ms is a metric for keygen. if it's just to fix openvm-prof, we should just filter it out from openvm-prof instead